app/helpers/status_helper.rb

```diff
@@ -22,7 +22,16 @@ def status_label(key)
   def status_panel(deploy)
     content = h deploy.summary
     if deploy.active?
-      content << content_tag(:ul, content_tag(:li, deploy.summary_for_process))
+      content << content_tag(:br)
+      content << "Started "
+      content << content_tag(
```

Comment:
don't we have some helper for that ? ... maybe create one so it's more descriptive ?